### ISSUE TITLE
Weird gray bar under videoplayer in fullscreen

### ISSUE THREAD

**Issue by rowbawts** [streamaserver/streama#705]
### Issue description
Full screen player has weird bar on bottom

### Steps to Reproduce

1. Play a movie
2. Put it in fullscreen
3. Weird bar

### Expected Behaviour

Player should just have black bars on top and bottom

### Actual Behaviour

Bottom of player is gray instead of black (only in fullscreen)

Fullscreen: 
![screenshot from 2018-12-02 22-30-16](https://user-images.githubusercontent.com/13012832/49351544-f5367100-f681-11e8-8b14-83d39bc04a09.png)
Not fullscreen: 
![screenshot from 2018-12-02 22-30-22](https://user-images.githubusercontent.com/13012832/49351554-ff586f80-f681-11e8-8fe3-8352985e1192.png)

### Environment Information

- **Operating System**: CentOS 7.5
- **Streama version**: v1.6.0-RC9
- **Custom streama build, describe customizations or provide link to fork (If Applicable)**: N/A
- **Container Version (If Applicable):** N/A

**Comment by HannahRy** [comment]
Very interesting. Can you test that this only happens in streama? Try grabbing the video file Strema is playing and drag it into a new tab and play that in fullscreen.

**Comment by rowbawts** [comment]
Unfortunately I just deleted it off of my local storage but I tried playing it in mpv before that and it worked as intended, and thanks for the quick reply!

> Very interesting. Can you test that this only happens in streama? Try grabbing the video file Strema is playing and drag it into a new tab and play that in fullscreen.

**Comment by rowbawts** [comment]
Ok so yeah the file plays normally in the browser

Fullscreen: 
![screenshot from 2018-12-02 22-49-42](https://user-images.githubusercontent.com/13012832/49352200-cff73200-f684-11e8-81ee-d301fc1f3ab1.png)

Not fullscreen: 
![untitled2](https://user-images.githubusercontent.com/13012832/49352212-d71e4000-f684-11e8-9cc1-2c39bfa4ca7b.jpg)

**Comment by HannahRy** [comment]
I'll try and recreate it sometime.

**Comment by rowbawts** [comment]
I'm pretty sure it has to do with the fact that I cropped out the bars in the video file when I ran it through Handbrake the other files that I did the same with don't exhibit this behavior but that's the only thing I can think of. If that is the issue it's still an issue because that shouldn't affect it.

**Comment by HannahRy** [comment]
I agree, if the video doesn't fill the display it should have black around whatever side needs. Having one black and one grey is very weird.

**Comment by f1reflyyyylmao** [comment]
I had this issue too, but I'm not sure how to replicate it either. I didn't cut any black bars with ffmpeg, although i did reencode the videos from h264 to webm. When the error occured i would exit fullscreen and re-enter it and eventually the bar would go black again. I'll report more information if i see it happening again.

**Comment by rowbawts** [comment]
@HannahRy I'm running into another issue I wanted to get on the discord but the invite is invalid can you give me a new one?

**Comment by f1reflyyyylmao** [comment]
Update: It just happened again for me. I was using firefox on windows to play a 4:3 episode on a 16:9 screen. The bar disappeared after i closed fullscreen to take a screenshot and didn't reappear after. I'm not sure if this is an issue with streama or firefox...

**Comment by EchoDev** [comment]
If you press the fullscreen button 5-10 times it happens you can reproduce the issue pretty easily.

If you press anywhere outside the window the grey bar disappears.
